assessments/intake-process.md

```diff
@@ -0,0 +1,34 @@
+# Security Assessment Priorities & Pipeline Intake Process
+
+SIG-Security has a volunteer team of subject matter experts and industry professionals dedicated to helping SIG-Security members, the TOC, and the larger CNCF community maintain an understanding of the current state of security in the cloud native ecosystem and helping cloud native projects succeed. 
+
+The following process describes how projects are prioritized for security assessments.
+
+# Authority
+
+* The [Security Assessment Facilitator](../governance/roles.md#security-assessment-facilitator) maintains the Assessment queue
+* A named chair provides oversight for the Security Assessment initiative. 
+This chair can adjust queue, and is also responsible for conveying TOC 
+instructions, when needed. (Sarah Allen is the current named Chair for Security Assessments.)
+
+# Pre-conditions
+
+* The project is either a CNCF project OR an assertion that the project is cloud native (any objection must be resolved before an assessment would be considered)
+* The project has identified a project lead and has a written self-assessment
```

Comment:
Recommend re-writing this as it doesn't cover potential incubation projects (implied maybe?)  something maybe to the effect of:
Projects must meet the following preconditions in order to be placed in queue for security assessment:
* existing CNCF project
* eligible for CNCF incubation and intention to join (link to CNCF eligibility)

Reply:
I think "an assertion that the project is cloud native (any objection must be resolved before an assessment would be considered)" lightly covers projects that have submitted proposals.  

I am intentionally allowing for us to review a project that does not plan to join the CNCF (but is cloud native).  I don't think we'll do this often (maybe never), but it could come up if a project is widely used by CNCF projects and is already part of some other foundation and it would benefit CNCF projects or end-users to have an assessment of that project.

Reply:
LGTM